core/src/avm2/globals/flash/net/URLVariables.as

```diff
@@ -1,9 +1,21 @@
 package flash.net {
     import flash.utils.escapeMultiByte;
+    import flash.utils.unescapeMultiByte;
     public dynamic class URLVariables {
-        // TODO: construct from String
-        // TODO: implement decode()
-        public function URLVariables() {}
+        public function URLVariables(str: String = "") {
```

Comment:
According to docs, default should be null. Probably doesn't *really* matter, but might as well
```suggestion
        public function URLVariables(str: String = null) {
```